src/coffee-script.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
```

Comment:
I figured this was tested via the fact that the test suite loaded at all. It seemed kinda weird to add a coffee-script spec that tests that another coffee-script file can be required.